docs_outdoornav_user_manual/features/navigation.mdx

```diff
@@ -0,0 +1,69 @@
+| **Path Smoothing**          | The path smoothing feature according to a specified turning radius. The default behaviour will generate point-to-point straight line paths.<br/><br/>Environment Variable: `ENABLE_PATH_SMOOTHER` (Default: `false`). |
+| **Path Shifting**           | The path shifting feature is designed to reduce the oscillation around the initial reference path if the UGV begins to deviate of said reference path. It is particularly useful for the Clearpath Robotics Warthog platform whose tires are incredibly pliable and results in unmodelled effects on the navigation.<br/><br/>Environment Variable: `ENABLE_PATH_SHIFTING` (Default: `false`). |
+| **Constrained Replanning**  | The constrained replanning feature restricts the area in which replanning paths can be generated. For example, if a `REPLANNING_CONSTRAINT` of 3.0 m is used, the UGV will not be allowed to replan a path that drives it more than 3.0 m from the initial path. This replanning area can be shown on the map over the connecting lines between waypoints. See [Constrained Replanning](/docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode#constrained_path) for further details.<br/><br/>Environment Variable: `ENABLE_CONSTRAINED_REPLANNING` (Default: `false`).<br/><br/>Use the `REPLANNING_CONSTRAINT` environment variable to modify the replanning constraint (in meters). |
+| **Stop Distance**           | The stop distance feature allows the UGV to stop a predefined distance away from obstacles. This is useful if a UGV cannot drive in reverse and needs the required room in front of it to replan around an obstacle.<br/><br/>Environment Variable: `ENABLE_STOP_DISTANCE` (Default: `false`)<br/><br/>Use the `STOP_DISTANCE` environment variable to modify the stop distance (in meters). Note that if the stop distance is larger than 4.5 meters for the Clearpath Robotics Jackal and Husky UGVs, or 10.0 meters for the Clearpath Robotics Warthog UGV, the computational load will increase and may cause a decrease in performance in the navigation. |
+| **Delay Compensation**      | The delay compensation feature is able to compensate for mechanical delay on UGVs where either the accelerator introduces delay into the linear velocity or the steering introduces delay in the angular velocity. This feature is not required for Clearpath Robotics UGVs as negligible delay is present in our system.<br/><br/>Environment Variable: `ENABLE_DELAY_COMPENSATION` (Default: `false`)<br/><br/>Use the `CONTROLLER_DELAY` environment variable to modify the amount of delay to be compensated (in milliseconds). |
+
+## Advanced Configuration

```

Comment:
Should we add an INFO notice to this section that modification of these parameters may affect system safety and possibly even that the user should only modify them in consultation with CPR?